src/models/CustomTabGroup.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    ViewColumn,
+    Tab as VSCodeTab
+} from 'vscode';
+
+/**
+	 * Represents a group of tabs. A tab group itself consists of multiple tabs.
+	 */
+export default interface CustomTabGroup{
+    /**
+     * Whether or not the group is currently active.
+     *
+     * *Note* that only one tab group is active at a time, but that multiple tab
+     * groups can have an {@link activeTab active tab}.
+     *
+     * @see {@link Tab.isActive}
+     */
+    readonly isActive: boolean;
```

Comment:
Does this mean the tab is selected from a UI point of view?

Reply:
to my understanding, yes